libraries/rush-lib/src/schemas/pnpm-lock-validation.schema.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Rush pnpm-lock-validation.schema.json config file",
+  "description": "This configuration file defines custom validation rules for pnpm-lock.yaml files in Rush projects. Each rule specifies a regular expression pattern that will be used to check the contents of pnpm-lock.yaml for compliance with project standards.",
+  "type": "object",
+  "properties": {
+    "pnpmLockValidationRules": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "ruleName": {
+            "type": "string",
+            "description": "A unique name for the validation rule."
+          },
+          "description": {
+            "type": "string",
+            "description": "A description of what the rule validates."
+          },
+          "pattern": {
+            "type": "string",
+            "description": "The regular expression pattern to use for validation."
+          }
+        },
+        "required": ["ruleName", "pattern"]
+      }
+    }
+  },
+  "required": ["pnpmLockValidationRules"]
```

Comment:
Suggest to call this setting something like `lockfilePolicyRules` or `pnpmLockfilePolicies`.  Rush uses the term "policy" to mean extra opt-in requirements that prevent common problems.  The term "validation" can mean the same thing, but generally "validation" is more about detecting actual mistakes or misconfigurations.